sources/platform/workflow/job/runsh.md

```diff
@@ -113,7 +113,7 @@ A description of the job YML structure and the tags available is in the [jobs se
   the behavior of the job.
     * `nodePool` --  The node pool on which the job should run. For detailed
       explanation, read about [node pools](/platform/management/subscription/node-pools/).
-    * `timeoutMinutes` -- Optional. This sets the maximum time, in minutes,
+    * `timeoutMinutes` -- Optional. This sets the maximum time(in minutes) after which you want your runSh job to timeout. If this is not provided then timeout set at nodePool, subscription or system level will be used, given preference in the same order.
```

Comment:
- `nodePool` -> `Node Pool`
- `subscription` -> Subscription
- default value of system level should be provided here